Week1/Week01.js

```diff
@@ -0,0 +1,48 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+});
+
+connection.connect();
+const database = 'create database if not exists world';
+const useDatabase = 'use world';
+
+const countries =
+  'create table country (name varchar(50), continent varchar(50), region varchar(50), surface_area int, independence_year date, population int, gnp int, gnp_old int, local_name varchar(50), government_form varchar(50), head_of_state text, capital varchar(50))';
+const cities =

```

Comment:
Be careful some values can not be int as they require floating point :) Could you fix them ? 
Otherwise your naming is perfect!

Reply:
fixed them :)